src/components/Home/home.js

```diff
@@ -1,5 +1,5 @@
 import React, { Fragment, useState } from 'react';
-import { Link, Redirect } from 'react-router-dom';
```

Comment:
I like that if you don't use a dependency you delete it to lose time in importing it.